include/roaring/portability.h

```diff
@@ -437,9 +437,16 @@ static inline int roaring_hamming(uint64_t x) {
 #define croaring_htobe64(x) OSSwapInt64(x)
 
 #elif defined(__has_include) && \
-    __has_include(<byteswap.h>)  // CROARING_IS_BIG_ENDIAN
+    __has_include(              \
+        <byteswap.h>)  && (defined(__linux__) || defined(__FreeBSD__))  // CROARING_IS_BIG_ENDIAN
```

Comment:
Is the line break intended or automated?
```suggestion
    __has_include(<byteswap.h>) && \
    (defined(__linux__) || defined(__FreeBSD__))  // CROARING_IS_BIG_ENDIAN
```

Reply:
We use automated formatting (clang-format).